src/Message/Communicate.cpp

```diff
@@ -35,6 +35,10 @@
 #include "ADIOS/ADIOS_config.h"
 #endif
 
+#ifdef HAVE_MPI
+#include "mpi3/shared_communicator.hpp"
```

Comment:
We only use the shared_communicator via the split_shared function.
This header inclusion seems not very logical.
It is probably a design issue in mpi3